tests/drivers/gpio/gpio_basic_api/src/test_callback_trigger.c

```diff
@@ -54,7 +54,7 @@ static int test_callback(int mode)
 	}
 
 	/* 2. configure PIN_IN callback and trigger condition */
-	rc = gpio_pin_configure(dev, PIN_IN, GPIO_INPUT);
+	rc = gpio_pin_configure(dev, PIN_IN, (GPIO_INPUT | PIN_IN_FLAGS));
```

Comment:
Has this test been looked at on platforms that do not have this flag?

Reply:
`PIN_IN_FLAGS` will be the GPIO flags of `in-gpios` node or zero if none, pls see https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/devicetree/gpio.h#L227